apps/ledger-live-mobile/src/reducers/LargeMover.ts

```diff
@@ -0,0 +1,25 @@
+import { handleActions } from "redux-actions";
+import type { Action, ReducerMap } from "redux-actions";
+
+export type LargeMoverState = {
+  tutorial: boolean;
+};
+
+export enum LargeMoverActionTypes {
+  SET_TUTORIAL = "LARGE_MOVER/SET_TUTORIAL",
+}
+
+export const LARGE_MOVER_INITIAL_STATE: LargeMoverState = {
+  tutorial: true,
+};
+
+const handlers: ReducerMap<LargeMoverState, LargeMoverState> = {
+  [LargeMoverActionTypes.SET_TUTORIAL]: (state, action) => ({
+    tutorial: (action as Action<LargeMoverState>).payload?.tutorial,
```

Comment:
in LargeMoverLandingPage you're calling `setTutorial(true)`, but isn't this handler expecting `setTutorial({tutorial: true})`? Also I don't think payload could be undefined